assets/src/domain/eventEditor/services/apollo/queries/tickets/useTickets.ts

```diff
@@ -4,7 +4,7 @@ import useCacheQuery from '../useCacheQuery';
 
 const useTickets = (): Array<Ticket> => {
 	const options = useTicketQueryOptions();
-	const { data } = useCacheQuery<TicketsList>(options);
+	const { data } = useCacheQuery<TicketsList>({ ...options, fetchPolicy: 'cache-first' });
```

Comment:
this is the fix for empty array of tickets?

Reply:
Yes